flows/actions/testdata/call_dialogflow.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
Can we add some test data? Seems to be basically the JSON definitions from a flow that has the Call DialogFlow flow step in there.